src/test/java/seedu/sellsavvy/logic/commands/ordercommands/AddOrderCommandTest.java

```diff
@@ -76,9 +78,31 @@ public void execute_duplicatePendingOrder_warningGiven() {
         assertCommandSuccess(addOrderCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_similarPendingOrder_warningGiven() {
+        // add something similar(but not identical) to the first order to the person's order list
+        String editedItemString = normalise(ABACUS.getItem().fullDescription);
+        Order toAdd = new OrderBuilder(ABACUS).withItem(editedItemString).build();
+        AddOrderCommand addOrderCommand = new AddOrderCommand(INDEX_FIRST, toAdd);
+
+        //ensures it is not equal to ABACUS
+        assertNotEquals(toAdd, ABACUS);
+
+        model.getFilteredPersonList().get(0).getOrderList().add(toAdd.createCopy());
+
+        Model expectedModel = model.createCopy();
+        Person personToAddUnder = expectedModel.getFilteredPersonList().get(0);
+        personToAddUnder.getOrderList().add(toAdd);
+        expectedModel.updateSelectedPerson(personToAddUnder);
+        String expectedMessage = String.format(MESSAGE_DUPLICATE_ORDER_WARNING
+                + AddOrderCommand.MESSAGE_ADD_ORDER_SUCCESS, personToAddUnder.getName(), Messages.format(toAdd));
+
+        assertCommandSuccess(addOrderCommand, model, expectedMessage, expectedModel);
+    }
+
     @Test
     public void execute_duplicateCompletedOrder_success() {
```

Comment:
Actually is there a point in testing this? We can't add a completed order into order list as far as i know.

Reply:
the purpose of this is to ensure that there is a duplcate with diff status no warning is thrown. I will add comment to help clarify and this code is by u in the first place...

Reply:
that works then